<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgRyMainWindow</class>
 <widget class="QMainWindow" name="ImgRyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imgRy | the image processor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(117, 117, 117);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnConvert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  color: rgb(255, 255, 255);
  background-color: transparent;
  border-radius: 0;
  border-width: 0 px;
}

QPushButton:hover:!pressed
{
  color: rgb(117, 117, 117);
}</string>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="traceViewer">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 170, 127);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDirPath">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 255, 255);
background-color: rgb(85, 170, 255)</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>d:\prv\pics\kepek\</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>18</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>path...</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 170, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="imgRatioSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>48</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="sliderPosition">
     <number>5</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="ratioLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>46</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 255, 255);
background-color: rgb(85, 170, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>46</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 170, 127);</string>
    </property>
    <property name="text">
     <string>Reduction ratio</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>btnConvert</zorder>
   <zorder>traceViewer</zorder>
   <zorder>lineEditDirPath</zorder>
   <zorder>imgRatioSlider</zorder>
   <zorder>ratioLabel</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
